src/backend/distributed/transaction/transaction_management.c

```diff
@@ -111,18 +111,18 @@ static bool MaybeExecutingUDF(void);
 
 
 /*
- * BeginOrContinueCoordinatedTransaction starts a coordinated transaction,
- * unless one already is in progress.
+ * UseOrKeepCoordinatedTransaction sets up the necessary variables to use
+ * a coordinated transaction, unless one is already in progress.
  */
 void
-BeginOrContinueCoordinatedTransaction(void)
+UseOrKeepCoordinatedTransaction(void)
```

Comment:
If we remove the other function, could we omit "OrKeep"? Use seems descriptive enough.

Reply:
Yeah I will remove the other function and remove `OrKeep`